### PR TITLE
Controllers spec

### PR DESCRIPTION
**Pull request by SFarida**
### Project requirements
- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/users_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+      # Check the http response for the correct placeholder text
+      # expect(response.body).to include('User Index Works')
+
+      # Check the http status of the response
+      expect(response.status).to eq(200)
+
+      # Check if the correct template was rendered
+      expect(response).to render_template('index')
+    end
```
Comment (Deepakdanger) [inline spec/users_controller_spec.rb]: - [ ] I would also like to see the tests that ensure the response body includes the correct placeholder for all the specs.

ex: `expect(response.body).to include("User index Works")`

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @SFarida, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- All tests are passing :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/users_controller_spec.rb (hunk 1 above)
Comment (SFarida) [reply spec/users_controller_spec.rb]: Hello @Deepakdanger sure that is possible let me update and push, am sorry for that, I actually added it but commented the lines after.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @SFarida  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉


## Highlight 👍 
Made sugged changes

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**